pkg/standalone/runfileconfig/run_file_config.go

```diff
@@ -41,3 +50,57 @@ type EnvItems struct {
 	Name  string `yaml:"name"`
 	Value string `yaml:"value"`
 }
+
+func (a *App) GetLogsDir() string {
+	logsPath := filepath.Join(a.AppDirPath, ".dapr", "logs")
+	if _, err := os.Stat(logsPath); errors.Is(err, os.ErrNotExist) {
+		os.MkdirAll(logsPath, 0o755)
```

Comment:
nit: is `os.Stat` needed here ? we can just do `os.MkdirAll`, it will create dirs if not present.?

Reply:
done